docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -136,4 +136,34 @@ Below topologies from ONNX open model zoo are fully supported on OpenVINO Execut
 | --- | --- | --- | --- | --- |
 | tiny_yolov2 | Yes | Yes | Yes | Yes* |
 
-*FPGA only runs in HETERO mode wherein the layers that are not supported on FPGA fall back to OpenVINO CPU.
\ No newline at end of file
+*FPGA only runs in HETERO mode wherein the layers that are not supported on FPGA fall back to OpenVINO CPU.
+
+## CSharp API
+
+To use csharp api for openvino execution provider create a custom nuget package.
+
+1. Windows
+
+Build a custom nuget package for windows.
+```
+.\build.bat --config Debug --build --use_openvino $Device --build_csharp
```

Comment:
Nit: maybe worth calling out that there will be 2 nugets: Managed and the "native" nuget and we need both for the target  c# app